src/components/confirm-icons.tsx

```diff
@@ -0,0 +1,72 @@
+import { Box, Button, Chip, IconButton, SvgIconTypeMap } from '@mui/material';
+import React, { useState } from 'react';
+import { useTranslator } from '../hooks';
+import CloseIcon from '@mui/icons-material/Close';
+import { OverridableComponent } from '@mui/material/OverridableComponent';
+
+export function ConfirmIcon(props: {
+  clickHandler: () => void;
```

Comment:
Rename this to `onConfirm`. This really should be `async` as well, but it looks like we're calling stop job via JL commands instead of simply awaiting the scheduler service methods. Refactoring those is outside the scope of this PR.